src/pages/activities/list/list.page.ts

```diff
@@ -107,19 +124,47 @@ export class ActivitiesListPage implements OnInit {
           },
           err => {
             loadingActivities.dismiss().then(() => {
-              loadingFailed.present();
+              if (ACTIVATE_TOAST) {
+                loadingFailed.present();
+              }
             });
           }
         )
   }
+
+  /**
+   * @TODO: remove this feature after development near complete
+   * Prompt user to skip loading to skip forced long wait of API
+   * @param {Function} cb callback if user choose to load API call
+   */
+  promptSkipLoading(cb: Function) {
```

Comment:
@trtshen now that this is developed, can the code be removed.
if so please work with @DamengRandom to remove.